### PR TITLE
Added recipe for nltk

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/nltk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+requirements:
```
Comment (jakirkham) [inline recipes/nltk/meta.yaml]: nit: Please add a newline between these sections.

File: recipes/nltk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - nltk.tokenize
+    - nltk.translate
+    - nltk.twitter
+
+about:
+  home: http://nltk.org/
+  license: Apache 2.0
+  license_file: LICENSE.txt
+  summary: 'Natural Language Toolkit'
+  license_family: Apache
```
Comment (jakirkham) [inline recipes/nltk/meta.yaml]: Maybe group the license info together.

**Comment by pmlandwehr** [comment]
Appveyor is ostensibly done; doing a close/reopen

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Comment by jakirkham** [comment]
@stevenbird @kmike @dhgarrette @alvations @fievelk @wroberts, just letting you know that we have added `nltk` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ( [`conda install -c conda-forge nltk`]( http://anaconda.org/conda-forge/nltk ) ). The maintenance burden here is light as all releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this feedstock (repo)]( https://github.com/conda-forge/nltk-feedstock ). If any of you have any interest in being added as a maintainer (to make new releases for instance), we would be more than happy to add any of you to the maintainers list. Thanks for your work on this great project. 😄

**Pull request merged**